models/mednist_gan/configs/metadata.json

```diff
@@ -0,0 +1,60 @@
+{
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
```

Comment:
Hi @ericspod , thanks for the updates. As for the `schema` here, it may still need to be modified, since it requires key `image` within `inputs`, but `latent` is used here.
There is not error raised here, since the `verify_metadata` printed the log rather than raised error here. The related place is just modified (see: https://github.com/Project-MONAI/MONAI/pull/4686).
Therefore, should we prepare a new `schema` here?

Reply:
The issue is that the input isn't an image but a latent vector of values so it wouldn't be correct to call it that. We're going to have other networks eventually that don't take images as input but instead have one or more inputs like a latent vector or 1D signals. What can we do to modify the schema to allow for arbitrarily-named inputs and outputs that's backwards compatible with the current schema?

Reply:
This will work for validation but then we aren't really checking anything now. Perhaps we're limited by what is possible in the schema but we would want to be able to say that the "properties" section contains one or more items of with arbitrary non-duplicate names having the structure given currently in "image". For now we can use this new schema to get this bundle finalised if anyone else is good with that.

Reply:
Hi @ericspod , according to our last meeting's discussion, I uploaded your schema in: https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_generator_20220718.json
could you please use this schema so far for this bundle? Then this PR can be merged. I will create a ticket about the new schema design, and we can enhance that part in the future.

Reply:
https://github.com/Project-MONAI/model-zoo/issues/65

Reply:
That's done, we do need to have a think about how a general schema will work.